index.js

```diff
@@ -20,7 +20,10 @@ var filesLoadedBeforeTrace = Object.keys(require.cache);
 
 // Load continuation-local-storage first to ensure the core async APIs get
 // patched before any user-land modules get loaded.
-require('continuation-local-storage');
+if (require('semver').satisfies(process.version, '<8') &&
```

Comment:
This is true. We can reorder things and add logic to remove `semver` from the list of modules loaded before trace. However, it seems highly unlikely that `semver` would take a dependency on module that would be problematic for us. Also, since it is required by our module instead of by the user, any issues it causes should manifest in our own tests.